.github/actions/generic-ci/action.yml

```diff
@@ -232,6 +232,7 @@ runs:
       run: |
         cmake --install . --component sdk
         cmake --install . --component configuration
+        cmake --install . --component colormaps
```

Comment:
add an install test ?

Reply:
How? Are we checking things like mimetype or config installation somewhere?  
I think testing it is the SB responsibility?

Reply:
How? Are we checking things like mimetype or config installation somewhere?  
I think testing it is the SB responsibility?

Reply:
We are not testing mimetypes, but we are testing config installation alright. See generic-ci/action.yml, look for `install`.

It is tested by the superbuild as well, but it is different, the superbuild is testing the shipped package, not the installation, which is a step further.

Reply:
found it, thanks
I extended the current test